lib/init/helptext.html

```diff
@@ -14,160 +14,135 @@ <h2>1. Graphical startup of GRASS GIS</h2>
 
 <i>For text-based startup see below.</i>
 <p>
-
-After <a href="grass7.html">launching</a> GRASS GIS, the startup screen will open:
+Compared to previous versions of GRASS, the concept of starting has completely
+changed in version 8. You will no longer find the graphical startup screen in
+this version. The Data tab, which takes care of organizing data, will become
+your new friend.
+</p>
 <p>
+After first GRASS GIS <a href="grass7.html">launch</a>, you will come to the
+<b>default Location</b>. This prepared template contains a map layer called "country_boundaries"
+showing a world map in WGS84 coordinate system:
+</p>
  
 <center>
-  <img src="grass_start.png" alt="[GRASS GIS start screen]">
+  <img src="grass_start.png" alt="[GRASS GIS after first startup]">
 </center>
 
-<h3><img src="circle_1.png" alt="[(1)]">&nbsp;Selecting the GIS Database directory</h3>
-
-GRASS data are stored in a directory referred to as DATABASE
-(also called "GISDBASE").  This directory has to be created
-with a file manager or the <tt>mkdir</tt> command, before starting to work
-with GRASS.  Within this DATABASE, the projects are organized
-by project areas stored in subdirectories called LOCATIONs.
-
-<h3><img src="circle_2.png" alt="[(2)]">&nbsp;Selecting the LOCATION (a project)</h3>
-
-A LOCATION is defined by its coordinate system, map projection and
-geographical boundaries. The subdirectories and files defining a
-LOCATION are created automatically when GRASS is started the first
-time with a new LOCATION. It is important to understand that each
-projection stays in its own LOCATION.
+<br>
 <p>
-See the "Location Wizard"&nbsp;<img src="circle_4.png" alt="[(4)]"> to
-easily create a new LOCATION from scratch from a geocoded file, by
-defining the parameters or by selecting the corresponding EPSG projection
-code.
+The main component of the Data tab is the Data Catalog. It uses a unique hiearchical
+structure consisting of Database <img src="grassdb.png" alt="[GRASS Database]">,
+Location <img src="location.png" alt="[Location]"> and Mapset <img src="mapset.png" alt="[Mapset]">.
+This concept of data organization is deeply rooted in the GRASS philosophy and
+you will meet it every time - does not matter whether you prefer a graphical
+user interface or a command line.
```

Comment:
I suggest removing it. Alternatively, you can add "you are doomed" at the end:)

Reply:
Isn't that "doomed" Vashek's idea? :-)
Ok, the reason why I've added this sentense was that without this sentence it looks a bit like "this data hiearchy structure is only important for GUI users"..

Reply:
I'm with Anna here. However, I see your point, Linda. Having removed the former lines 27-31, if we also remove this, then it seems that the data organization is a GUI thing. 

I'd suggest to remove this sentence from here, but describe the grass data organization earlier in the doc, maybe recycling a bit of what was already in lines 27-31. In that way it's clear that the data structure is generally relevant and data catalog is just a way to display this structure and data.

Reply:
Yep, you are talking about my first idea. :-) But then I thought that it is better to start with what the user sees immediately  after startup (this is the default location) and then explain the data hierarchy on the example of this default location.

Reply:
@lindakladivova not my idea, but I asked Grassy the Hungry Cow about the tone of the changes and this what she said: 

```
 __________________________
| It's all gloom and moom! |
 --------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
_\|/__\|/__\|/_ ||     ||  _\|/__\|/_
```